bin/galsim.py

```diff
@@ -95,6 +95,17 @@ def parse_args():
             '-m', '--module', type=str, action='append', default=None, 
             help='python module to import before parsing config file')
         parser.add_argument(
+            '-p', '--profile', action='store_const', default=False, const=True,
+            help='output profiling information at the end of the run')
+        parser.add_argument(
+            '-n', '--njobs', type=int, action='store', default=1, 
+            help='set the total number of jobs that this run is a part of. ' + 
+            'Used in conjunction with -j (--job)')
+        parser.add_argument(
+            '-j', '--job', type=int, action='store', default=1,
+            help='set the job number for this particular run. Must be in (1 .. njobs). ' +
```

Comment:
This is really trivial, but I'd recommend `[1, njobs]` for the range notation.  The use of regular parenthesis to mean a closed interval (even though it's obvious what you mean with a little thought) threw me off for just a second.